model.py

```diff
@@ -0,0 +1,50 @@
+from __main__ import app
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy.orm import backref
+from werkzeug.security import generate_password_hash, check_password_hash
+
+db = SQLAlchemy(app)
+tk=SQLAlchemy(app)
```

Comment:
remove it, and use only db.
Doesn't require to initialize each time for each table.